fpga_interchange/converters.py

```diff
@@ -434,7 +437,8 @@ def from_reader(message,
             continue
 
         if field_proto_data.ref_annotation is not None:
-            reference_fun = lambda value: reader.reference_value(field_proto_data.ref_annotation, value, root, parent)
+            reference_fun = lambda value: reader.reference_value(
```

Comment:
I'm still slightly confused by these autoformatting changes, but it' passing CI so presumably this format is OK too?